lib/widgets/message_list.dart

```diff
@@ -505,8 +514,7 @@ class MarkAsReadWidget extends StatelessWidget {
               ),
               onPressed: () => _handlePress(context),
```

Comment:
It is possible to click on it again while previous `markNarrowAsRead` is still in progress, click should be disabled while it's in `loading` state, not sure if this should be different issue+PR or just a commit in this PR.

Reply:
cc @gnprice